eng/Version.Details.xml

```diff
@@ -20,5 +20,9 @@
       <SourceBuild RepoName="runtime" />
       -->
     </Dependency>
+    <Dependency Name="Microsoft.CodeAnalysis.CSharp" Version="4.0.0-2.21323.33">
+      <Uri>dotnet/roslyn</Uri>
```

Comment:
I know darc can fix the missing sha but not sure it can fix the Uri so probably safer to use `https://github.com/dotnet/roslyn`